contracts/gateway/AxelarAmplifierGateway.sol

```diff
@@ -19,11 +19,20 @@ contract AxelarAmplifierGateway is BaseAmplifierGateway, BaseWeightedMultisig, U
 
     struct AxelarAmplifierGatewayStorage {
         mapping(bytes32 => bool) rotations;
+        address operator;
```

Comment:
other name suggestions? this overlaps with the `operator` from the original gateway which had a different meaning. `rotationOperator` doesn't feel too different as well.

Reply:
`rotationRecoveryAdmin` ?